app/views/feature_flags/index.html.slim

```diff
@@ -0,0 +1,32 @@
+.govuk-grid-column-two-thirds
+  h1.govuk-heading-xl = t(".feature_flags")
+  = govuk_table do |table|
+    ruby:
+      table.caption(size: "m", text: t(".time_based"))
+      table.body do |body|
+        body.row do |row|
+          row.cell(text: t(".flag"), header: true)
+          row.cell(text: t(".enabled"), header: true)
+        end
+        FeatureFlags.time_dependant.each do |flag|
+          body.row do |row|
+            row.cell(text: flag)
+            row.cell(text: FeatureFlags.enabled?(flag) ? t("generic.yes_choice") : t("generic.no_choice"))
+          end
+        end
+      end
+  = govuk_table do |table|
+    ruby:
+      table.caption(size: "m", text: t(".static"))
+      table.body do |body|
+        body.row do |row|
+          row.cell(text: t(".flag"), header: true)
+          row.cell(text: t(".enabled"), header: true)
+        end
+        FeatureFlags.static.each do |flag|
+          body.row do |row|
+            row.cell(text: flag)
+            row.cell(text: FeatureFlags.enabled?(flag) ? t("generic.yes_choice") : t("generic.no_choice"))
+          end
+        end
+      end
```

Comment:
Not a blocker just a question - I know this isn't meant for the public, but am I right in saying this would be difficult for screen readers? We haven't specified table headings, body and row as per component guidelines.

Reply:
It's a good question. I wasn't sure so I added a spec to `spec/system/accessibility_spec.rb`, which does automated accessibility best-practice checking. It didn't complain about this page which makes me think that those additional fields, while good practice strictly speaking, aren't necessary to make the page accessible for screen readers. The automated check isn't perfect, so it could have missed something tho.

Reply:
Fair enough and glad it's had some automated checks on it at least - here's the [suggested standards](https://www.w3.org/WAI/tutorials/tables/one-header/), I checked against.